packages/code/src/Code.tsx

```diff
@@ -288,14 +274,22 @@ function Code({
     },
     className,
     getScrollShadowStyle(scrollState, mode),
+    { [singleLineWrapperStyle]: !isMultiline },
+    {
+      [css`
+        &:focus {
+          outline: none;
+        }
+      `]: !showFocus,
+    },
   );
 
   const renderedSyntaxComponent = (
     <Syntax
       showLineNumbers={showLineNumbers}
       darkMode={darkMode}
       language={language}
-      highlightLines={highlightLines}
+      highlightLines={highlightLines || []}
```

Comment:
Shouldn't need this, since we highlightLines will default to [] already

Reply:
You'd think so, but I was hitting an exception before adding this. Can't remember exactly what it was but if you feel strongly, I can probably reproduce it.

Reply:
I tried to repro locally and am not seeing an issue 🤔

Reply:
I think it was an edge case when switching between different options in the highlight lines knob. Let me see if I can reproduce it. I remember this just being an edge case I hit when testing the component manually.

Reply:
I couldn't reproduce, so I'm removing this change.